app/reducers/index.js

```diff
@@ -1,11 +1,13 @@
 import { combineReducers } from 'redux'
 import { routerReducer } from 'react-router-redux'
 
-import { theme } from './theme_reducer.js'
+import { theme } from './theme.js'
+import { subscribeState } from './subscribe.js'
 
 const rootReducer = combineReducers({
     router: routerReducer,
-    theme
+    theme,
+    subscribeState
```

Comment:
confused on the variable naming- Are they all reducers? Is subscribeState also a reducer?